### PR TITLE
Remove NPQ Google Drive integration

### PR DESCRIPTION
**Pull request by peteryates**
DfE no longer uses Google Drive and this integration is redundant.

### CODE REVIEW

**Review by leoapost — APPROVED** [review]

**Review by leandroalemao — APPROVED** [review]
Just a question regarding the 131 EligibilityImport records in the database in prod.. are we gonna use it for something? If we won't so all good.

**Comment by peteryates** [comment]
Google Drive was deactivated a year or more ago so anything in there is now well out of date.

**Review by cwrw — COMMENTED** [review]
looking good, can we remove the `google-apis-drive_v3` gem as well?

File: .env.development [hunk 1]
```diff
@@ -1,6 +0,0 @@
-GOOGLE_CLIENT_ID=
```
Comment (cwrw) [inline .env.development:1]: could we also remove those from `.env.template`?

**Pull request merged**